File: egolint/tests/fixtures/scalafix/GreetingService.scala
package egohygiene.fixtures

object GreetingService {
  def formatGreeting(name: String): String = {
    val normalizedName = name.trim

    if (normalizedName.isEmpty) {
      "Hello."
    } else {
      s"Hello, $normalizedName."
    }
  }
}

File: egolint/tests/fixtures/rubocop/greeting_service.rb
# frozen_string_literal: true

module EgoHygiene
  module Fixtures
    class GreetingService
      def format_greeting(name)
        normalized_name = name.strip

        return 'Hello.' if normalized_name.empty?

        "Hello, #{normalized_name}."
      end
    end
  end
end
